File: backend/src/main/java/com/licenseair/backend/domainModel/OrderNotifyModel.java
package com.licenseair.backend.domainModel;

import java.util.List;
import lombok.Data;

import java.lang.String;
import java.sql.Timestamp;
import java.lang.Integer;
import java.sql.Timestamp;

/**
* Created by licenseair.com
* OrderNotify
*/
@Data()
public class OrderNotifyModel {
  public Long id = null;

  /**
  *
  */
  public String method = null;

  /**
  * 是否删除
  */
  public Integer deleted = null;

  /**
  *
  */
  public String sign = null;

  /**
  *
  */
  public String trade_no = null;

  /**
  *
  */
  public String charset = null;

  /**
  *
  */
  public String auth_app_id = null;

  /**
  *
  */
  public String out_trade_no = null;

  /**
  *
  */
  public Timestamp timestamp = null;

  /**
  *
  */
  public String app_id = null;

  /**
  *
  */
  public String version = null;

  /**
  *
  */
  public String seller_id = null;

  /**
  *
  */
  public String total_amount = null;

  /**
  *
  */
  public String sign_type = null;

  public Timestamp created_at = null;

  public Timestamp updated_at = null;

}

File: generator/output/ts-model/order-notify.model.ts
class OrderNotifyModel {
  public id: number = 0;

  // method 
  public method?: string;

  // deleted 是否删除
  public deleted: boolean = false;

  // sign 
  public sign?: string;

  // trade_no 
  public trade_no?: string;

  // charset 
  public charset?: string;

  // auth_app_id 
  public auth_app_id?: string;

  // out_trade_no 
  public out_trade_no?: string;

  // timestamp 
  public timestamp?: string;

  // app_id 
  public app_id?: string;

  // version 
  public version?: string;

  // seller_id 
  public seller_id?: string;

  // total_amount 
  public total_amount?: string;

  // sign_type 
  public sign_type?: string;

  public created_at: string;

  public updated_at: string;
}

const OrderNotifyModelFields = {
  id: "id",
  method: "method",
  deleted: "deleted",
  sign: "sign",
  trade_no: "trade_no",
  charset: "charset",
  auth_app_id: "auth_app_id",
  out_trade_no: "out_trade_no",
  timestamp: "timestamp",
  app_id: "app_id",
  version: "version",
  seller_id: "seller_id",
  total_amount: "total_amount",
  sign_type: "sign_type",
  created_at: "created_at",
  updated_at: "updated_at"
};

export {OrderNotifyModel, OrderNotifyModelFields};
